djautotask/sync_rest.py

```diff
@@ -271,3 +291,130 @@ def _assign_field_data(self, instance, json_data):
     def get_page(self, next_url=None, *args, **kwargs):
         kwargs['conditions'] = self.api_conditions
         return self.client.get_contacts(next_url, *args, **kwargs)
+
+
+# ParentSynchronizer: using SOAP API
+class TicketSynchronizer(SyncRecordUDFMixin, Synchronizer, ParentSynchronizer):
+    client_class = api.TicketsAPIClient
+    model_class = models.TicketTracker
+    udf_class = models.TicketUDF
+
+    related_meta = {
+        'companyID': (models.Account, 'account'),
+        'status': (models.Status, 'status'),
+        'assignedResourceID': (models.Resource, 'assigned_resource'),
+        'priority': (models.Priority, 'priority'),
+        'queueID': (models.Queue, 'queue'),
+        'projectID': (models.Project, 'project'),
+        'ticketCategory': (models.TicketCategory, 'category'),
+        'ticketType': (models.TicketType, 'type'),
+        'source': (models.Source, 'source'),
+        'issueType': (models.IssueType, 'issue_type'),
+        'subIssueType': (models.SubIssueType, 'sub_issue_type'),
+        'assignedResourceRoleID': (models.Role, 'assigned_resource_role'),
+        'billingCodeID': (models.AllocationCode, 'allocation_code'),
+        'contractID': (models.Contract, 'contract'),
+        'contactID': (models.Contact, 'contact'),
+    }
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        request_settings = DjautotaskSettings().get_settings()
+        self.completed_date = (timezone.now() - timezone.timedelta(
+                hours=request_settings.get('keep_completed_hours')
+            )).strftime('%Y-%m-%dT%H:%M:%S.%fZ')
+        self.api_conditions = [
```

Comment:
I think there is something wrong with the API query conditions. Running `./manage.py atsync` should be a partial sync and only return records that have changed since the last sync job but I'm seeing all Ticket records returned on every sync.

Reply:
Another good point! Thanks